timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -85,6 +105,10 @@ class AbstractGluonTSModel(AbstractTimeSeriesModel):
 
     gluonts_model_path = "gluon_ts"
     gluonts_estimator_class: Type[GluonTSEstimator] = None
+    use_feat_static_cat: bool = False
+    use_feat_static_real: bool = False
+    use_feat_dynamic_real: bool = False
+    feat_static_cat_cardinality: List[int] = []
```

Comment:
We will infer these from `train_data` for all models (i.e., set to True if some of the static features available), but they will only be passed down to the model for `DeepAR` and `MQCNN`